src/components/User/Assets/AssetRequests/components/AssetRequestList.vue

```diff
@@ -75,9 +75,9 @@
             <!-- eslint-disable max-len -->
             <span
               class="app-list__cell"
-              :title="CREATE_ASSET_REQUEST_STATES[snakeToCamelCase(asset.state)].text"
+              :title="CREATE_ASSET_REQUEST_STATES[snakeToCamelCase(asset.state)].codeVerbose | assetRequestStatesFilter"
```

Comment:
Using filter must look like something this: `asset.state | assetRequestStatesFilter`

Reply:
done